styles/Main.css

```diff
@@ -228,32 +229,33 @@ div.favourites label {
   order: -1;
   flex-grow: 1;
 }
-.Gallery {
+.gallery {
   display: grid;
   grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 10px;
 }
-.Gallery img {
+.gallery img {
   display: block;
-  padding: 5px;
   width: 100%;
   height: 100%;
   object-fit: cover;
 }
-.Gallery img:hover {
+.gallery img:hover {
   scale: 1.1;
 }
-.Row-span-2 {
-  grid-row: 1 / span 2;
```

Comment:
Using just `span 2` allows the image to take its intrinsic starting position and then span 2 rows/cols from there